### PR TITLE
Update intro.

### PR DESCRIPTION
**Pull request by Awjin**
WDYT of this addition?

Fixes https://github.com/sass/sass-site/issues/531

### CODE REVIEW

**Comment by nex3** [comment]
I like the idea of replacing "variables" with a more modern CSS feature, but I worry that "modules" without any other context might lead people to think that Sass supports something like [CSS Modules](https://github.com/css-modules/css-modules). Maybe "style re-use"? Or is that too abstract?

**Comment by Awjin** [comment]
It might be too abstract compared to the other goodies listed. Hmm. How about "scoped imports"?

**Comment by nex3** [comment]
Feels weird to call that out since it's something we're actively deprecating. Maybe just "custom functions"?

**Comment by jina** [comment]
can just remove the word instead of replacing it.

File: source/guide.html.haml [hunk 1]
```diff
@@ -27,7 +27,7 @@ introduction: >
 
     CSS on its own can be fun, but stylesheets are getting larger, more
     complex, and harder to maintain. This is where a preprocessor can help.
-    Sass lets you use features that don't exist in CSS yet like variables,
+    Sass lets you use features that don't exist in CSS yet like modules,
     nesting, mixins, inheritance and other nifty goodies that make writing CSS
     fun again.
```
Comment (jina) [inline source/guide.html.haml:32]: While we are in here, can we also change the last phrase? "make [noun] [adjective] again" has been ruined for me by a certain public figure.

**Review by jina — CHANGES_REQUESTED** [review]

File: source/guide.html.haml (hunk 1 above)
Comment (jina) [reply source/guide.html.haml:32]: I also very much dislike the notion that CSS isn't fun. I love writing CSS.

Comment (Awjin) [reply source/guide.html.haml]: I think the original intent was that maintaining thousands lines of pure CSS isn't necessarily fun, but I get what you're saying. Made some edits.

Comment (jina) [reply source/guide.html.haml]: thank you! ❤️

**Review by jina — APPROVED** [review]

**Pull request merged**